misk/src/test/resources/misk/hibernate/docker-compose.yaml

```diff
@@ -0,0 +1,11 @@
+version: '3'
+services:
+  vitess:
+    image: "vitess/base:latest"
+    command: "../../../bin/vttestserver -alsologtostderr -port 27000 -web_dir=web/vtctld/app -web_dir2=web/vtctld2/app -mysql_bind_host=0.0.0.0 -schema_dir=schema  -keyspaces=movies,actors --num_shards=2,1"
```

Comment:
the implied working directory here doesn’t feel right. No guidance on what to do instead

Reply:
Yeah that's not entirely pretty but it's not our docker image and there's strength in being able to just use the one that's provided by the open source project. Maybe an absolute path is a bit better.

I'm sure we're going to iterate on this for a bit until we get it just right.

Reply:
An absolute path is a lot better